source/faq.txt

```diff
@@ -131,7 +131,11 @@ How do I view and modify my Privacy Settings?
 ---------------------------------------------
 
 To view and modify your |compass| privacy settings, from the top-level
-menu click :guilabel:`Help`, then :guilabel:`Privacy Settings`.
+menu:
+
+1. Click :guilabel:`Help`
```

Comment:
[suggestion] Lists require similar punctuation for each list item. I'd recommend adding a period to #1 and #2 since they are both complete sentences.

Reply:
Updated.